src/text-field-components.css.ts

```diff
@@ -102,13 +102,18 @@ export const fullWidth = sprinkles({
     width: '100%',
 });
 
-export const field = sprinkles({
-    overflow: 'hidden',
-    border: 'regular',
-    display: 'flex',
-    borderRadius: vars.borderRadii.input,
-    position: 'relative',
-});
+export const field = style([
+    sprinkles({
+        overflow: 'hidden',
+        border: 'regular',
```

Comment:
instead of setting border to regular and overriding the borderColor, I'd just set the border. Alternatively, we could define a new border type for inputs in [sprinkles](https://github.com/Telefonica/mistica-web/blob/4302740e75928802d0bf51e1995925f00eb9ab2e/src/sprinkles.css.ts#L64)

Reply:
I included new type in sprinkles to manage input borders from one side